Tests/XCUITests/BookmarkingTests.swift

```diff
@@ -281,9 +282,10 @@ class BookmarkingTests: BaseTestCase {
         // Check that it appears in Bookmarks panel
         navigator.goto(LibraryPanel_Bookmarks)
         waitForExistence(app.tables["Bookmarks List"], timeout: 5)
-        app.tables["Bookmarks List"].cells.staticTexts["Example Domain"].swipeLeft()
+        app.tables["Bookmarks List"].cells.staticTexts["Example Domain"].press(forDuration: 1)
```

Comment:
I'm not sure if the gesture change is an update for both M1 and Intel. Swipe left no longer does anything. We should use the context menu to remove a bookmark.

Reply:
The swipe gesture works on Intel, you can add an if condition until that is fixed on M1, so we do a swipe on Intel and long press on M1